news/management/commands/crawl_news.py

```
@@ -0,0 +1,151 @@
+
+                # CrawlJob 상태 확인
+                try:
+                    crawl_job.refresh_from_db()
+                    self.stdout.write(f"CrawlJob 상태: {crawl_job.status}")
+                    if crawl_job.error_message:
+                        self.stdout.write(
+                            self.style.ERROR(
+                                f"CrawlJob 오류 메시지: {crawl_job.error_message}"
+                            )
+                        )
+                except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)